pom.xml

```diff
@@ -1472,6 +1473,25 @@
         <groupId>org.scalatest</groupId>
         <artifactId>scalatest-maven-plugin</artifactId>
       </plugin>
+      <!-- Build the JARs-->
```

Comment:
Can this say something more useful? Maybe like:

```
<!-- Build test-jar's for all projects, since some projects depend on tests from others -->
```

Otherwise it might be hard for someone to understand what is going on here.